data/changelog.txt

```diff
@@ -160,6 +181,10 @@ Add favorite category to spell casting menu
 Allow scanning several books into ereader at once
 Add milling info to milleable items
 Add spell class selecter to spell casting menu
+'Learning is disabled' message on disabled skills
+Swaps UI widgets with ones that have proper labels
+Update gun mod removal UI to use inventory menu instead of a prompt; prevent removal of gun mods with other mods installed on it
+Show the mass of vitamins in food items
```

Comment:
```suggestion
'Learning is disabled' message on disabled skills
Update gun mod removal UI to use inventory menu instead of a prompt; prevent removal of gun mods with other mods installed on it
Show the mass of vitamins in food items
```